roles/st2mistral/tasks/main.yml

```diff
@@ -92,8 +92,8 @@
 
 - name: Configure mistral.conf settings
   # Ansible nested loop to iterate through a hash of hashes
-  include: config.yml _conf_section_name={{ _conf_section.key }} _conf_options={{ _conf_section.value }}
-  with_dict: "{{ st2mistral_config }}"
+  import_tasks: config.yml _conf_section_name={{ _conf_section.key }} _conf_options={{ _conf_section.value }}
```

Comment:
`include_tasks` with `loop`
To be forwards compatible with 2.7 (weird I know) 
Please move `_conf_*` vars to a `vars` block.
https://docs.ansible.com/ansible/devel/porting_guides/porting_guide_2.7.html#include-tasks-import-tasks-inline-variables

Reply:
will fix this two parts after discussion